backend/src/main/java/com/benine/backend/ServerController.java

```diff
@@ -130,6 +130,11 @@ private void startupDatabase() {
         e.printStackTrace();
       }
     }
+    try {
+      database.checkCameras();
```

Comment:
You are checking the camera's before they are loaded from the config file?
So there are no camera's to compare with yet.
Camera's are loaded from the config in line 74 of this file.